src/e2e/java/teammates/e2e/cases/e2e/AdminAccountsPageE2ETest.java

```diff
@@ -0,0 +1,96 @@
+package teammates.e2e.cases.e2e;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.AccountAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.e2e.pageobjects.AdminAccountsPage;
+import teammates.e2e.util.BackDoor;
+
+/**
+ * SUT: {@link Const.WebPageURIs#ADMIN_ACCOUNTS_PAGE}.
+ */
+public class AdminAccountsPageE2ETest extends BaseE2ETestCase {
+
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/AdminAccountsPageE2ETest.json");
+        removeAndRestoreDataBundle(testData);
+    }
+
+    @Test
+    public void testAll() {
+
+        String googleId = "AAMgtE2eT.instr2";
+
+        ______TS("verify loaded data");
+
+        AppUrl accountsPageUrl = createUrl(Const.WebPageURIs.ADMIN_ACCOUNTS_PAGE)
+                .withParam(Const.ParamsNames.INSTRUCTOR_ID, googleId);
+        AdminAccountsPage accountsPage = loginAdminToPage(accountsPageUrl, AdminAccountsPage.class);
+
+        AccountAttributes account = getAccount(googleId);
+        accountsPage.verifyAccountDetails(account);
+
+        ______TS("action: remove instructor from course");
+
+        String courseId = "AAMgtE2eT.CS2103";
+        String email = "AAMgtE2eT.instr2@gmail.tmt";
+
+        InstructorAttributes instructor = getInstructor(courseId, email);
+        verifyPresentInDatastore(instructor);
+        accountsPage.clickRemoveInstructorFromCourse(courseId);
+        accountsPage.verifyStatusMessage("Instructor is successfully deleted from course \"" + courseId + "\"");
+        verifyAbsentInDatastore(instructor);
+
+        ______TS("action: remove student from course");
+
+        StudentAttributes student = BackDoor.getStudent(courseId, email);
+        verifyPresentInDatastore(student);
```

Comment:
Same for `getStudent` and actually we do not have to import `BackDoor` here because of the `getStudent` method in `BaseTestCaseWithBackDoorApiAccess`